week1/project/hackyourweather/src/components/cityCard.js

```diff
@@ -0,0 +1,32 @@
+import cityWeather from "../city-weather";
+
+const CityCard = () => {
```

Comment:
Check the naming of your component `CityCard` implies that this one component but it is actually a list.

```suggestion
const CityCardList = () => {
```